azure.go

```diff
@@ -330,6 +380,8 @@ func (b *azureAuthBackend) getAzureSettings(ctx context.Context, config *azureCo
 	}
 	settings.ClientSecret = clientSecret
 
+	settings.IdentityTokenAudience = config.IdentityTokenAudience
+
```

Comment:
One small nit, we probably should set the token ttl here as well
```suggestion
settings.IdentityTokenTTL = config.IdentityTokenTTL
```

Reply:
Resolving since this was added in as part of the context removal PR!